Longest_Common_Subsequence/Longest_Common_Subsequence.php

```diff
@@ -0,0 +1,80 @@
+<?php 
+
+// Dynamic Programming implementation of LCS problem 
+
+// Returns length of LCS for X[0..m-1], Y[0..n-1] 
+function lcs( $X, $Y, $m, $n ) 
+{ 
+	$L = array_fill(0, $m + 1, 
+		array_fill(0, $n + 1, NULL)); 
```

Comment:
Please ensure only 4 spaces indentation here. It is more here.

Reply:
Okay. Sorry for that. I have edited the file. Please review again. Thank you